src/datasources/account/account.datasource.ts

```diff
@@ -0,0 +1,294 @@
+import { Inject, Injectable } from '@nestjs/common';
+import * as postgres from 'postgres';
+import { IAccountDataSource } from '@/domain/interfaces/account.datasource.interface';
+import { AccountDoesNotExistError } from '@/datasources/account/errors/account-does-not-exist.error';
+import {
+  Account as DomainAccount,
+  EmailAddress,
+} from '@/domain/account/entities/account.entity';
+import { Subscription as DomainSubscription } from '@/domain/account/entities/subscription.entity';
+
+interface Account {
+  id: number;
+  chain_id: number;
+  email_address: string;
+  safe_address: string;
+  signer: string;
+  verified: boolean;
+  verification_code: string | null;
+  verification_code_generated_on: Date | null;
+  verification_sent_on: Date | null;
+  unsubscription_token: string;
+}
+
+interface Subscription {
+  id: number;
+  key: string;
+  name: string;
+}
+
+@Injectable()
+export class AccountDataSource implements IAccountDataSource {
```

Comment:
Something to consider (in a separate PR) is ensuring all of the saved addresses are checksummed.